### ISSUE TITLE
[Feedback]Automount USB drive in settings toggle does not work

### ISSUE THREAD

**Issue by bmentink** [IceWhaleTech/CasaOS#1043]
### Description

I set the "Automount USB drive" switch off in settings, as I want to mount my own drives with UUID, however the OS still mounts the drive on /media, even with the switch off.

### Additional Information

	 - OS: linux
	 - CasaOS Version: 0.4.3
	 - Disk Total: 27712 
	 - Disk Used: 8634 
	 - System Info: [Debian GNU/Linux 11   \l Bit:64 Mem:3763M Core:4 Linux CasaOS 5.10.0-22-amd64 #1 SMP Debian 5.10.178-3 (2023-04-22) x86_64 GNU/Linux]
	 - Remote Version: 0.4.3
	 - Browser: Chrome 
	 - Version: 109.0.0.0

**Comment by bmentink** [comment]
This is a "Bug" not feedback!

**Comment by raller1028** [comment]
Yes this problem does exist, but when you turn it on and off again, the switch works fine

**Comment by InflatedBoi** [comment]
> Yes this problem does exist, but when you turn it on and off again, the switch works fine

Hi there !
I'm encountering the same issue. Is there any workaround to disable the auto-mount ?

Additional Information:
- CasaOS v0.4.4.3
- OS: Debian GNU/Linux 12 (bookworm) **aarch64**
- Host: Raspberry Pi 5 Model B Rev 1.0 (8Go version)
- SSD 1,7To,  FS type ext4
